cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go

```diff
@@ -278,6 +278,8 @@ type HostPathMount struct {
 	MountPath string `json:"mountPath"`
 	// Writable controls write access to the volume
 	Writable bool `json:"writable,omitempty"`
+	// PathType is the type of the HostPath.
+	PathType string `json:"pathType,omitempty"`
```

Comment:
Where is this defaulted, and what happens if it's an empty string?

Reply:
@timothysc The default value is an empty string. And it will create a volume like this:
```
volumes:
- hostPath:
      path: /foo/bar
      type: ""
```
And this will have the same behavior as the volume in which we do not specify the `type` of `hostPath`.

Reply:
/cc @liztio  

B/c it's affecting config.  You're basically adding a string for a v1-type.  Why wouldn't you have the type itself?